src/import/FSHImporter.ts

```diff
@@ -753,46 +760,57 @@ export class FSHImporter extends FSHVisitor {
       ? this.visitVsComponentFrom(ctx.vsComponentFrom())
       : {};
     if (ctx.vsFilterList()) {
-      ctx
-        .vsFilterList()
-        .vsFilterDefinition()
-        .forEach(filterDefinition => {
-          try {
-            filters.push(this.visitVsFilterDefinition(filterDefinition));
-          } catch (e) {
-            logger.error(e, {
-              location: this.extractStartStop(filterDefinition),
-              file: this.currentFile
-            });
-          }
+      if (!from.system) {
```

Comment:
Above, you do:
```ts
if (from.system) {
  // process it...
} else {
  // log an error...
}
```
 Here you do the reverse order:
```ts
if (!from.system) {
  // log an error...
} else {
  // process it...
}
```

It's not wrong, but it feels inconsistent since they both are doing the same thing at a high level.

Reply:
In addition, even though each error is essentially saying the same thing (that you must state the system), the two messages are worded very differently.